ingeniamotion/communication.py

```diff
@@ -37,6 +38,47 @@
 FILE_EXT_LFU = ".lfu"
 
 
+class RegisterUpdateCallbackModifier:
+    """Class that injects the servo alias to the register
+    update callback.
+
+     Args:
+         alias: Servo alias.
+         callback: Register update callback.
+
+    """
+
+    instances: Dict[str, "RegisterUpdateCallbackModifier"] = {}
+
+    def __new__(
+        cls, alias: str, callback: Callable[[str, Servo, Register], None]
+    ) -> "RegisterUpdateCallbackModifier":
+        if alias in cls.instances:
+            return cls.instances[alias]
+
+        obj = super().__new__(cls)
+
+        cls.instances[alias] = obj
```

Comment:
With this method, you can only subscribe to one callback by alias. I'm not sure if we will need to subscribe to it more than once.

Reply:
I'm aware, I think I can tweak it to allow multiple callbacks per servo.

Reply:
Done